source/plugin/scheduler.rst

```diff
@@ -67,24 +71,24 @@ Using the ``TaskBuilder``, you can specify other, optional properties, as descri
 |                 |                         | it should be descriptive and aid users in debugging your plugin.         |
 +-----------------+-------------------------+--------------------------------------------------------------------------+
 
-Last, submit the task to the scheduler using ``TaskBuilder#submit(Object plugin)``.
+Lastly, submit the task to the scheduler using ``Task.Builder#submit(Object plugin)``.
 
 And that's it! To summarize, a fully functional scheduled task that would run asynchronously every 5 minutes after an
 initial delay of 100 milliseconds could be built and submitted using the following code:
 
 .. code-block:: java
 
-	import org.spongepowered.api.service.scheduler.Task;
+    import java.util.concurrent.TimeUnit;    
+    
+    Scheduler scheduler = game.getScheduler();
+    Task.Builder taskBuilder = scheduler.createTaskBuilder();
 
-	SchedulerService scheduler = game.getScheduler();
-	TaskBuilder taskBuilder = scheduler.createTaskBuilder();
-
-	Task task = taskBuilder.execute(new Runnable() {
-		public void run() {
-			logger.info("Yay! Schedulers!");
-		}
-	}).async().delay(100, TimeUnit.MILLISECONDS).interval(5, TimeUnit.MINUTES)
-        .name("ExamplePlugin - Fetch Stats from Database").submit(plugin);
+    Task task = taskBuilder.execute(new Runnable() {
```

Comment:
Could convert this to a lambda expression as it's neater. Also could show that you can use a `Consumer<Task>` here instead.